src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,56 @@
+
+.nav {
+  &__list {
+    display: flex;
+
+    list-style: none;
+  }
+
+  &__item {
+    margin: 0 10px;
+  }
```

Comment:
Set margins for one direction, not for both. It's more clear and also you can remove it from one child, not for both first and last